src/server/api/routers/question.ts

```diff
@@ -41,50 +41,58 @@ export const questionRouter = createTRPCRouter({
     });
   }),
   getAllReducedInfinite: publicProcedure
-    .input(z.object({ cursor: z.string().optional(), limit: z.number().min(1).max(100).nullable(), titleFilter: z.string().optional() }))
-    .query(
-      async ({input, ctx}) => {
-        const limit = input.limit ?? 50;
-        const totalCount = await ctx.prismaMongo.question.count({
-          where: input.titleFilter ? {
-            OR: [
-              {
-                title: {
-                  contains: input.titleFilter
-                },
-              },
-              {
-                body: {
-                  contains: input.titleFilter
-                }
-              }
-            ]
-          } : undefined,
-        });
-        const items = await ctx.prismaMongo.question.findMany({
-          take: limit + 1,
-          where: input.titleFilter ? {
-            title: {
-              contains: input.titleFilter
+    .input(
```

Comment:
I think this change breaks the existing code in codespace, shld return nextCursor as well?